source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,44 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-01-26
+
+### Highlights
+
+#### Channel Sidebar General Availability
+ - The new channel sidebar is enabled system-wide by default. Mattermost now gives users flexibility to organize channels and direct messages into custom, collapsible sidebar categories. Users now gain full personalization of their sidebar to improve productivity, reduce clutter and focus on what matters. Features include custom, collapsible sidebar categories, unread channel management, and new channel menus.
```

Comment:
Do we want to clarify that what was once the experimental sidebar is now the sidebar everyone gets by default?

Reply:
@cwarnermm not in this case, since for the purposes of cloud users it was always turned on by default and users were not able to turn it off